insights/tests/client/connection/test_checkin.py

```diff
@@ -63,7 +65,7 @@ def test_canonical_facts_request(get_proxies, post, get_canonical_facts, rm_conf
     **{"return_value.status_code": codes.CREATED}
 )
 @patch("insights.client.connection.InsightsConnection.get_proxies")
-def test_canonical_facts_request_cleaned(get_proxies, post, get_canonical_facts, rm_conf):
+def test_canonical_facts_request_cleaned(get_proxies, post, get_canonical_facts, fetch_system_by_machine_id, rm_conf):
```

Comment:
If the above was true, the function signature would be different.

```suggestion
def test_canonical_facts_request_cleaned(get_proxies, post, get_canonical_facts, fetch_system_by_machine_id, init_session, rm_conf):
```

Just saying to finish the story, nothing related to your code again.